framework/src/org/checkerframework/framework/type/TypeFromTypeTreeVisitor.java

```diff
@@ -40,7 +40,7 @@
  *
  * @see org.checkerframework.framework.type.TypeFromTree
  */
-class TypeFromTypeTreeVisitor extends TypeFromTreeVisitor {
+class TypeFromTypeTreeVisitor extends TypeFromExpressionVisitor {
```

Comment:
Why this change?

Reply:
This was necessary because `getAnnotatedTypeFromTypeTree` was possibly called on an expression tree.
This is no longer necessary and I undid this.